content/docs/operating/configuration.md

```diff
@@ -278,14 +278,17 @@ CAUTION: Kubernetes SD is in beta: breaking changes to configuration are still
 likely in future releases.
 
 Kubernetes SD configurations allow retrieving scrape targets from
-[Kubernetes'](http://kubernetes.io/) REST API. By default, this discovers nodes
-and appropriately annotated services so that metrics from both cluster
-components and deployed applications can be scraped. This will create multiple
-target groups: one for all nodes with each node as a target; and one for each
-service containing each service endpoint as a target.
+[Kubernetes'](http://kubernetes.io/) REST API. By default, this discovers
+masters, nodes and appropriately annotated services so that metrics from both
+cluster components and deployed applications can be scraped. This will create
```

Comment:
Ha! I'm here now! No commas in "both cluster components and deployed applications" (putting commas there would be wrong).

See also http://english.stackexchange.com/questions/107182/how-to-use-commas-around-the-word-both